prow_config.yaml

```diff
@@ -19,9 +39,29 @@ workflows:
     job_types:      
       - postsubmit
     include_dirs:
-    - pkg/*
-    - cmd/*
-    - test/*
+    - pkg/api/operators/*
+    - pkg/api/health/*
+    - pkg/api/v1alpha1/*
+    - pkg/common/v1alpha1/*
+    - pkg/controller/v1alpha1/*
+    - pkg/db/v1alpha1/*
+    - pkg/earlystopping/v1alpha1/*
+    - pkg/manager/v1alpha1/*
+    - pkg/manager/modelstore/*
+    - pkg/suggestion/v1alpha1/*
+    - pkg/ui/*
+    - cmd/earlystopping/*
+    - cmd/katib-controller/v1alpha1/*
+    - cmd/manager/v1alpha1/*
```

Comment:
both v1alpha1 and v1alpha2 share same Dockerfile for some sub dir, we should include the Dockerfile in two workflows

Reply:
Make sense, addressed. Thanks